README.md

```diff
@@ -48,12 +48,6 @@ This repository contains a sample [Microsoft Teams bot](https://docs.microsoft.c
 
    - The ngrok account owner that setup the domain should have this information
```

Comment:
This is true, so then maybe we can make a note of this? Something along the lines of "to run the bot on localhost via the bot emulator (instead of ngrok), run `npm start` outside of the dev container"